### DISCUSSION TITLE
[2024-11-11] Libs and Engine Team Demo Day

### DISCUSSION THREAD

**Discussion by GregoryTravis** [enso-org/enso#11521] [Weekly Demo Day]
- James: Excel before 1900 and AWS signed requests. #11373
- James: Expand and improve `pretty` for core data types, vector and table. #11438
- James: Experiment in GUI Styling #11465
- James: Improved Google Analytics integration #11484
- James: Support for 1904 date format. #11496
- James: New NumberParser for Table parsing #11499
- Radosław: Infer method calls on known types and warn about nonexistent methods #11399
- Radosław: Fixing small bugs uncovered by type checker #11422
- Radosław: Enso_File integration update: Multi-part upload and presigned URL for download #11440
- Radosław: Add widget for `name_filter` #11455
- Radosław: Initial implementation of `Data.read_many` #11490
- Greg: Fix `Float.parse` benchmark regression #11402
- Greg: Clean up SQLite file after the SQLite file tests have run. #11416
- Greg: Add Warnings whenever a floating-point value is used as a Dictionary key #11417
- Greg: Create Regex benchmarks #11456
- Greg: Make `round_integer` work for positive decimal places #11457
- Greg: Create Regex benchmarks #11503
- Greg: Create Regex benchmarks #11504
- Ad: Enable SQLServer Sort Feature and associated tests #11379
- Ad: SQLServer enable filter #11471
- Ad: First draft #11495
- Ad: Refactor Internal_Core #11506
- Ad: Make SQLServer join work #11511
- Martha: Add data quality indicators to table Viz #11307
- Martha: Extend Selection Menu to allow text selection #11352
- Martha: Draft- Table viz column header tooltip extended to add data quality stats #11520
- Hubert: Don't cancel aborted jobs immediately #11375
- Hubert: Fix logging in CLI #11472
- Hubert: More logs to debug suspend issue #11518
- Hubert: Fix Logger's name in stdlib #11519
- Pavel: Convert `Array_Like_Helpers.map` to a builtin to reduce stack size #11363
- Pavel: Fix Meta.get_qualified_type_name when run as single file #11401
- Pavel: Fix displaying of host values in chrome devtools #11468
- Pavel: Only single native image is built at a time #11497
- Pavel: --jvm tries to find Java executable system-wide. #11500
- Pavel: More IR mini passes #11501
- Dmitry: Metadata should not depend on absolute text spans #11390
- Dmitry: Reload insight script on file change #11415
- Dmitry: Dependency tracking between nodes is too coarse grained #11428
- Dmitry: Add compression to the metadata code snapshot #11470
- Dmitry: Build Ydoc on CI #11517

**Comment by JaroslavTulach** [comment]
- #11366
   - goal to **represent** `Table` with a _single column_ as `Table&Column`
   - in progress: #11482
- Fixed a nasty bug in `IR.copy`: #11513
    - #11512
- Towards `if_then_else` **not being a function** -#11365
    - #11419
    - #11464
    - #11486
    - #11491
- towards: #10121
    - `test/Geo_Tests`: #11394
    - `Base_Internal_Tests`: #11403
    - Huge issues with `Standard.Image`: #11483
    - Similar problems with `Standard.Database` - SQLite, Postgress binaries
- some stability work: #11393
- preparing for  http://javadays.cz/ conference
